.circleci/config.yml

```diff
@@ -0,0 +1,86 @@
+version: 2
+jobs:
+  build:
+    parallelism: 4
+    working_directory: ~/wp-calypso
+    docker:
+      - image: circleci/node:8.9.4-browsers
+        environment:
+          CIRCLE_ARTIFACTS: /tmp/artifacts
+          CIRCLE_TEST_REPORTS: /tmp/test_results
+
+    steps:
+      - checkout
+
+      - run:
+          name: Create Directories for Results and Artifacts
+          command: |
+            mkdir -p $CIRCLE_ARTIFACTS
+            mkdir -p $CIRCLE_TEST_REPORTS
+
+      - restore_cache:
+          key: v1-{{ .Branch }}-{{ checksum "package.json" }}
+
+      - run: npm install
+
+      - save_cache:
+          key: v1-{{ .Branch }}-{{ checksum "package.json" }}
```

Comment:
It should be `checksum "npm-shrinkwrap.json"` if I'm understanding this correctly.

Reply:
should this also be there for `restore_cache`?

Reply:
Yes, since in general the dependencies can change even though `package.json` stays the same. The `shrinkwrap` is the only source of truth.

Reply:
That reminds me that we should look into using `npm ci` over `npm install` when it becomes available in the dockerfile/circle config, as it only looks at the shrinkwrap/lockfile

http://blog.npmjs.org/post/171556855892/introducing-npm-ci-for-faster-more-reliable